### ISSUE TITLE
[BUG] Saudi Arabia GP race result missing position related values

### ISSUE THREAD

**Issue by Casper-Guo** [theOehrly/Fast-F1#547]
### Describe the issue:

Maybe related to #545 

The following columns are not populated in the session results for the 2024 Saudi Arabia GP:

- Position
- GridPosition
- ClassifiedPosition
- Q1
- Q2
- Q3
- Time
- Points

### Reproduce the code example:

```python
import fastf1

session = fastf1.get_session(2024, 2, "R")
session.load(telemetry=False, weather=False, messages=False)
result = session.results
print(result.isna().sum())
```

### Error message:

```shell
DriverNumber           0
BroadcastName          0
Abbreviation           0
DriverId               0
TeamName               0
TeamColor              0
TeamId                 0
FirstName              0
LastName               0
FullName               0
HeadshotUrl            0
CountryCode            0
Position              20
ClassifiedPosition     0
GridPosition          20
Q1                    20
Q2                    20
Q3                    20
Time                  20
Status                 0
Points                20
dtype: int64

count     20
unique     1
top         
freq      20
Name: ClassifiedPosition, dtype: object
```

**Comment by Casper-Guo** [comment]
@mdsakalu I haven't looked too deeply but do you think this might also be fixed by #546?

**Comment by theOehrly** [comment]
Yes, it's the same problem.

(Just noting that Q1, Q2, Q3 are expected to not be populated in a race session.)
